lenovo/thinkpad/t14s/default.nix

```diff
@@ -7,6 +7,11 @@
     ../../../common/pc/laptop/acpi_call.nix
```

Comment:
This profile already imports `amd`. Should that not be moved to the amd profile?

Reply:
I'm not sure if I understand you correctly. Do you mean we should move line 6 to `lenovo/thinkpad/t14s/amd/default.nix`?

Reply:
Given you changes, I think we are on the same page.